src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Providers/GetAzureProviderCmdlet.cs

```diff
@@ -89,5 +91,38 @@ public override void ExecuteCmdlet()
                 this.WriteObject(providers, enumerateCollection: true);
             }
         }
+
+        private PSResourceProvider[] ListPSResourceProviders()
+        {
+            var providers = this.ResourceManagerSdkClient.ListResourceProviders(
+                providerName: this.ProviderNamespace, 
+                listAvailable: this.ListAvailable);
+
+            if (string.IsNullOrEmpty(this.Location))
+            {
+                return providers
+                    .Select(provider => provider.ToPSResourceProvider())
```

Comment:
SG

---
In reply to: [65991290](https://github.com/Azure/azure-powershell/pull/2380#discussion_r65991290) [](ancestors = 65991290)